SU-SWS/stanford_profile
# D8CORE-4118 Remove layout builder settings on other display modes

# READY FOR REVIEW

# Summary
- Remove unwanted layout builder settings on card and search index display modes.
- When choosing a different layout option on the node page this fixes the card display in views.

# Need Review By (Date)
- 4/30

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. import configs
3. create 2 basic pages and tag them with the same "Basic Page Type" term
   - One of them choose a "Full Width" layout
   - populate the metadata image and description for both nodes
4. Create another basic page and use the list paragraph type
   - Choose the basic page view with card display
   - Use your taxonomy term from earlier as the argument
 5. View the list
 6. Verify both page cards display the image and the meta description.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
